moped-database/metadata/tables.yaml

```diff
@@ -3662,7 +3662,6 @@
       permission:
         check: {}
         columns:
-          - capitally_funded
```

Comment:
i decided to remove this column from the DB. we're not using anywhere and can generate it on the fly (if eCapris project id is not null then it is capitally funded).

Reply:
🙏🏼